content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -297,7 +300,29 @@ You can also find the options to share the signup form in the form editor by cli
 
 ## Edit or Remove Contacts
 
-During this early phase of beta, you can only view and edit contact details from the All Contacts page. In the future you'll be able to view and edit contact details from lists. You'll also be able to delete contacts. 
+To manage your contact lists more efficiently, you can edit and remove contact lists from the Contacts page. 
+
+*To edit a contact:*
+
+1. Navigate to the Contacts page and search for the contact you want to edit.
+1. Click the contact you wish to edit. This takes you to the contact details page.
+1. To edit any of the profile fields, click on the *Edit* icon to the right of the field.
+1. Once you have finished making all of your changes, click **Save**. 
+
+*To delete a contact from your account:*
+
+1. Navigate to the Contacts page and search for the contact you want to edit.
```

Comment:
My understanding is that based on the 4/10 release the instrux to delete a contact entirely are simpler: 

1. Navigate to the Contacts page and search for the contact you want to edit.
1. Click the delete icon in the Action column to permanently delete the contact from your account. This cannot be undone.

Reply:
@negarzipan